tests/admin.article.test.js

```diff
@@ -0,0 +1,178 @@
+import chai from 'chai';
+import '@babel/polyfill';
+import chaiHttp from 'chai-http';
+import status from '../src/helpers/constants/status.codes';
+import server from '../src/index';
+import error from '../src/helpers/constants/error.messages';
+import signup from '../src/helpers/tests/signup';
+
+chai.use(chaiHttp);
+chai.should();
+
+const fakeAdmin = signup();
+
+let userToken = '';
+const slug = 'How-to-create-sequalize-seeds';
+const password = 'password23423';
+const adminUrl = '/api/admin/article';
+const Article = {
+  id: 1,
+  reason: 'this content exploits the minors',
+
+  title: 'How to create sequalize seeds',
+  description: 'How to set dummy data automatically',
+  body:
+    "Suppose we want to insert some data into a few tables by default. If we follow up on previous example we can consider creating a demo user for User table.To manage all data migrations you can use seeders. Seed files are some change in data that can be used to populate database table with sample data or test data.Let's create a seed file which will add a demo user to our User table.",
+  slug: 'How-to-create-sequalize-seeds',
+  coverImage: 'default.jpeg',
+  tagList: ['postgres', 'express', 'sequelize'],
+  createdAt: '2019-07-08T17:15:05.710Z',
+  updatedAt: '2019-07-08T17:15:05.710Z',
+  Category: { name: 'LOVE' },
+  User: {
+    firstName: 'Alain',
+    lastName: 'Burindi',
+    profileImage: 'defaultAvatar.jpg'
+  }
+};
+
+describe('Article', () => {
+  it('should login the moderetor to get the token', (done) => {
```

Comment:
use the same method as you used up there when you were trying to get the token of the fakeAdmin

Reply:
I have to log-in two different users, and the helper login only one user.
and the second user should be the admin or moderator @MCFrank16